load-balancer/src/it/java/org/apache/phoenix/end2end/LoadBalancerEnd2EndIT.java

```diff
@@ -139,6 +142,37 @@ public void testSingletonPropertyForLoadBalancer(){
         Assert.assertTrue(" the load balancer is not singleton",loadBalancer == anotherloadBalancerRef );
     }
 
+    @Test
+    public void testZkConnectString() {
+        // server port is different from the port which is set by user
```

Comment:
Nit Technically, this is a UT, but we tend not worry about the distinction much.